### ISSUE TITLE
Add extra settings in Reveal.initialize

### ISSUE THREAD

**Issue by chxp** [yjwen/org-reveal#291]
I want to custom the reveal.js-menu (https://github.com/denehyg/reveal.js-menu) in Reveal.initialize(), So How to do it in org-reveal like this:

```
Reveal.initialize({
	// ...

	menu: {
		side: 'left',
		numbers: false,
		autoOpen: true,
		...		
	},

});
```

**Comment by yjwen** [comment]
Add a new option `#+REVEAL_EXTRA_INITIAL_JS` so that such code can be embedded by

`#+REVEAL_EXTRA_INITIAL_JS: menu: {slide: 'left', numbers:false, autoOption:true}`

A customizable variable `org-reveal-extra-initial-js` also exists for the same purpose.

**Comment by dangom** [comment]
Unfortunately `dc946bc` just broke ox-reveal. 

There is a comma too much after the introduction of `(if-format "%s,\n" (plist-get info :reveal-extra-initial-js))`:

```html
Reveal.initialize({

controls: true,
progress: true,
history: false,
center: true,
slideNumber: 'c',
rollingLinks: false,
keyboard: true,
overview: true,

theme: Reveal.getQueryHash().theme, // available themes are in /css/theme
transition: Reveal.getQueryHash().transition || 'default', // default/cube/page/concave/zoom/linear/fade/none
transitionSpeed: 'default',
multiplex: {
    secret: '', // null if client
    id: '', // id, obtained from socket.io server
    url: '' // Location of socket.io server
},
,
```

**Comment by yjwen** [comment]
Just fixed by 63a63d5 . Sorry about that typo.
